l10n_ec_withhold/wizard/wizard_create_sale_withhold.py

```diff
@@ -30,6 +30,15 @@ def default_get(self, fields):
         if len(invoices.partner_id) > 1:
             raise UserError(_("Choose a multiple invoices of a same customer"))
 
+        is_paid = False
+        for i in invoices:
+            if i.payment_state == "paid":
+                is_paid = True
+        if is_paid:
```

Comment:
Una manera mas eficiente y legible seria
```suggestion
        if any(invoice.payment_state == "paid" for invoice in invoices):
```